.gitignore

```diff
@@ -12,6 +12,7 @@
 node_modules/
 /npm-debug.log
 /.idea/
+sf.env
```

Comment:
Maybe name this "salesforce.env" just to be explicit.

Reply:
Good idea.